### PR TITLE
feat(app): add NetworkSettings to RobotSettings Dashboard

### PR DESCRIPTION
**Pull request by koji**
# Overview
This will add NetworkSettings screen to RobotSettings Dashboard.
Calling network setup function will be added in a following PR. 
This PR uses the custom hook https://github.com/Opentrons/opentrons/pull/12095

[design](https://www.figma.com/file/AoTLAYuWawlaWItB1umOjr/OT-3-On-Device-Display?node-id=1797%3A30368&t=msYu5aBeHLFdFyUn-0)

# Test Plan
Requirements
- connected status and ssid name for WiFi
connect a dev kit to a wifi network and Robot Dashboard -> Settings(Navigation) -> Network Settings 

- connected status for Ethernet
plug an ethernet cable into a dev kit and Robot Dashboard -> Settings(Navigation) -> Network Settings

For this PR USB is descoped since not implemented the detect function right now.

# Changelog
- Add NetworkSettings component

# Review requests

# Risk assessment
low

### CODE REVIEW

**Comment by koji** [comment]
- [x] export `Chip` as a component Mel confirmed that we would need this as one of atoms component
- [x] use `Chip` component https://github.com/Opentrons/opentrons/pull/12090
- [x] use `useNetworkConnection` https://github.com/Opentrons/opentrons/pull/12095
- [x] add tab functionality for rendering each setting component https://github.com/Opentrons/opentrons/pull/12098
- [x] merge the above PRs into this PR
- [x] solve conflicts
- [x] fix connection status on RobotSettingsDashboard button

**Comment by koji** [comment]
- [ ] merge edge

File: app/src/organisms/OnDeviceDisplay/RobotSettingsDashboard/NetworkSettings.tsx [hunk 1]
```diff
@@ -3,34 +3,163 @@ import { useTranslation } from 'react-i18next'
...
+  buttonTitle,
+  buttonBackgroundColor,
+  iconName,
+  chipType,
+  chipText,
+  chipIconName,
+  networkName,
+}: NetworkSettingButtonProps): JSX.Element {
+  return (
+    <Btn
```
Comment (brenthagen) [inline app/src/organisms/OnDeviceDisplay/RobotSettingsDashboard/NetworkSettings.tsx]: minor point, but what does this outer button do? seems like maybe the inner button onclick should be out here. also, a button within a button is not valid HTML: `Warning: validateDOMNesting(...): <button> cannot appear as a descendant of <button>.`

**Review by brenthagen — APPROVED** [review]
📶

**Pull request merged**